main.py

```diff
@@ -1,6 +1,8 @@
 import os
 
 import discord
+import logging
```

Comment:
Is this stuff all in default python lib? If not, can you run `make freeze` to update the dependencies in requirements.txt?

Reply:
logging is part of python lib, will do this anyways